apps/ledger-live-mobile/src/components/Nft/NftViewerBackground.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FC } from "react";
+import { View, StyleSheet, Image, Animated } from "react-native";
+import Svg, { Defs, LinearGradient, Stop, Rect } from "react-native-svg";
+import { useTheme } from "@react-navigation/native";
+
+// Number of scrollY pixels to animate 1 -> 0
+const OPACITY_ANIMATION_PERIOD = 200;
+const BLUR_RADIUS = 45;
+
+interface Props {
+  readonly src: string;
+  readonly scrollY: Animated.Value;
+}
+
+const NftViewerBackground: FC<Props> = ({ src, scrollY }) => {
+  const { dark } = useTheme();
+  const gradientColor = dark ? "#131214" : "#fff";
```

Comment:
`#131214` doesn't exist in the theme colors. I also see `useTheme` imported from `react-native` and styled components in various places. What's the consensus here for doing this kind of thing?

Reply:
I believe `#131214` is `background.main` for dark and `#ffffff` for light.